auth.go

```diff
@@ -95,6 +102,16 @@ func New(opts Options) (Service, error) {
 		return nil, err
 	}
 
+	buttonTpl, err := readTemplate(opts.ButtonTpl)
+	if err != nil {
+		return nil, err
+	}
+
+	if len(opts.Scopes) == 0 {
```

Comment:
If you don't want the button to be shown no scopes will be provided, this should not be checked. If the scopes are 0 the feature should be disabled